index.bs

```diff
@@ -531,11 +533,9 @@ The suborigin of a resource is the value computed by the following algorithm:
   </li>
 
   <li>
-    If the Content Security Policy of the resource contains a valid
-    <a>suborigin directive</a> in the <a
-    href="https://www.w3.org/TR/CSP2/#policy-syntax">directive list</a>
-    [[!CSP2]], then let `suborigin-namespace` be the
-    `directive-value`.
+    If the HTTP response of the resource contains a valid
+    <pre>suborigin</pre> header, then let `suborigin-namespace` be the
```

Comment:
<pre>suborigin</pre> -> [suborigin]

or if you want to keep the code look, [`suborigin`] should also work.

Bikeshed FTW!